composer.json

```
@@ -21,7 +21,7 @@
         "psr/http-server-middleware": "^1.0",
         "psr/http-server-handler": "^1.0",
         "ramsey/uuid": "^3.0 || ^4.0",
-        "doctrine/orm": "^2.7, <2.9",
+        "doctrine/orm": "^2.9.2",
```

```suggestion
        "doctrine/orm": "^2.9.3",
```
Is broken before that patch-level release